build.gradle

```diff
@@ -262,13 +263,22 @@ project(':ambry-frontend') {
     }
 }
 
+project(':ambry-validation-service') {
+    dependencies {
+        compile project(':ambry-tools')
+        compile project(':ambry-utils')
+        compile "com.codahale.metrics:metrics-core:$metricsVersion"
+    }
+}
+
 subprojects.each { subproject -> evaluationDependsOn(subproject.path)}
 
 task allJar(type: Jar, dependsOn: subprojects.assemble) {
     manifest {
         attributes 'Implementation-Title': 'Ambry',
                    'Main-Class': 'com.github.ambry.server.AmbryMain'
     }
+    zip64 true
```

Comment:
Got the following error on build. Hence the fix.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':allJar'.
> archive contains more than 65535 entries.
  
  To build this archive, please enable the zip64 extension.
  See: https://docs.gradle.org/2.12/dsl/org.gradle.api.tasks.bundling.Zip.html#org.gradle.api.tasks.bundling.Zip:zip64